SPECS/etcd/etcd-3.4.13.spec

```diff
@@ -0,0 +1,121 @@
+Summary:        A highly-available key value store for shared configuration
+Name:           etcd
+Version:        3.4.13
+Release:        1%{?dist}
+License:        ASL 2.0
+URL:            https://github.com/etcd-io/etcd/
+Group:          System Environment/Security
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+#Source0:       https://github.com/etcd-io/%{name}/archive/v%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+Source1:        etcd.service
+
+BuildRequires:  golang >= 1.13
+BuildRequires:  git
+
+%description
+A highly-available key value store for shared configuration and service discovery.
+
+%prep
+%setup -q
+
+%build
+# Turn off auto moduling. golang 1.13 does not automatically consider the vendor folder (it does as of 1.14).
+# To successfully build, manually hydrate the go package cache (GOPATH) with the included vendor folder and
+# etcd's source code before invoking the build script.
+export GO111MODULE=off
+
+%define OUR_GOPATH %{_topdir}/.gopath
+mkdir -p "%{OUR_GOPATH}/vendor" "%{OUR_GOPATH}/etcd_src/src/go.etcd.io"
+export GOPATH=%{OUR_GOPATH}/vendor:%{OUR_GOPATH}/etcd_src
+
+ln -s "%{_builddir}/%{name}-%{version}/vendor" "%{OUR_GOPATH}/vendor/src"
+ln -s "%{_builddir}/%{name}-%{version}" "%{OUR_GOPATH}/etcd_src/src/go.etcd.io/etcd"
+
+./build
+
+%install
+install -vdm755 %{buildroot}%{_bindir}
+install -vdm755 %{buildroot}/%{_docdir}/%{name}-%{version}
+install -vdm755 %{buildroot}/lib/systemd/system
+install -vdm 0755 %{buildroot}%{_sysconfdir}/etcd
+install -vpm 0755 -T etcd.conf.yml.sample %{buildroot}%{_sysconfdir}/etcd/etcd-default-conf.yml
+
+chown -R root:root %{buildroot}%{_bindir}
+chown -R root:root %{buildroot}/%{_docdir}/%{name}-%{version}
+
+mv %{_builddir}/%{name}-%{version}/bin/etcd %{buildroot}%{_bindir}/
+mv %{_builddir}/%{name}-%{version}/bin/etcdctl %{buildroot}%{_bindir}/
+mv %{_builddir}/%{name}-%{version}/README.md %{buildroot}/%{_docdir}/%{name}-%{version}/
+mv %{_builddir}/%{name}-%{version}/etcdctl/README.md %{buildroot}/%{_docdir}/%{name}-%{version}/README-etcdctl.md
+mv %{_builddir}/%{name}-%{version}/etcdctl/READMEv2.md %{buildroot}/%{_docdir}/%{name}-%{version}/READMEv2-etcdctl.md
+
+install -vdm755 %{buildroot}/lib/systemd/system-preset
+echo "disable etcd.service" > %{buildroot}/lib/systemd/system-preset/50-etcd.preset
+
+cp %{SOURCE1} %{buildroot}/lib/systemd/system
+install -vdm755 %{buildroot}/var/lib/etcd
+
+%post   -p /sbin/ldconfig
+
+%postun -p /sbin/ldconfig
+
+%clean
+rm -rf %{buildroot}/*
+
+%files
+%license LICENSE
+%{_bindir}/etcd*
+/%{_docdir}/%{name}-%{version}/*
+/lib/systemd/system/etcd.service
+/lib/systemd/system-preset/50-etcd.preset
+%dir /var/lib/etcd
```

Comment:
Nit: I'm not sure if we use that anywhere, but `/var/lib` can be represented with `%{_sharedstatedir}`.

Reply:
changed in both etcd spec